stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -213,6 +213,21 @@ func posixPipe() -> (readFD: CInt, writeFD: CInt) {
   return (fds[0], fds[1])
 }
 
+#if !SWIFT_STDLIB_HAS_ENVIRON
+@_silgen_name("_NSGetEnviron")
+func _NSGetEnviron() -> UnsafeMutablePointer<UnsafeMutablePointer<UnsafeMutablePointer<CChar>?>>
+
+var environ: UnsafeMutablePointer<UnsafeMutablePointer<CChar>?> {
+  #if os(macOS)
+  return _NSGetEnviron().pointee
+  #elseif os(Linux) || os(Windows)
+  return __environ
+  #else
```

Comment:
Sorry, I think that we got something confused here ...

```suggestion
  #elseif os(Windows)
  return __p_environ().pointee
  #elseif os(Linux)
  return __environ
  #else
```